### DISCUSSION TITLE
Healthcheck to use in Kubernetes configuration

### DISCUSSION THREAD

**Discussion by hyrefeldt-ourstudio** [riverqueue/riverui#156] [Q&A] [answered]
Hi! I'm looking into deploying the UI on Kubernetes. Is there a healthcheck endpoint to use for the Kubernetes liveness probe? I can't find any info about that in the docs.

**Comment by brandur** [comment, marked as answer]
Try [`GET /api/health-checks/minimal`](https://github.com/riverqueue/riverui/blob/master/CHANGELOG.md#added-2).

**Reply by hyrefeldt-ourstudio** [reply]
Thanks! That worked fine 🙌
